homeworks/Alexey.Berkut_AlBerkut/homework_5/innerCopy.js

```diff
@@ -0,0 +1,8 @@
+const obj = { b: 'c', d: { e: 'f' } };
+
+function innerCopy(input) {
+    const objCopy = Object.assign({}, input);
+    return objCopy;
```

Comment:
this will not perform a deep copy of object, read about variables as references to objects